samples/drivers/watchdog/boards/s32z270dc2_rtu0_r52.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		watchdog0 = &swt0;
+	};
+};
+
+&swt0 {
+	status = "okay";
```

Comment:
Why not enable this in the board as well?

Reply:
I have enabled it in the board files and this is not needed here.